src/core/common_path.py

```diff
@@ -0,0 +1,6 @@
+from pathlib import Path
```

Comment:
@vasylboliuk please don't forget to add docstring at the model level, describing what this module is used for. This is a PEP8 requirement (don't forget to add an empty line between the module docstring and the first import"

Reply:
Done